### ISSUE TITLE
Can't debug anymore the office 365 workbench since 1.1.0 ?

### ISSUE THREAD

**Issue by seaSide974** [SharePoint/sp-dev-docs#634]
#### Category
- [ ] Question
- [ ] Typo
- [? ] Bug
- [ ] Additional article idea

#### Expected or Desired Behavior
Runs Office 365 workbench in debug mode

#### Observed Behavior
I run gulp serve
Start debugging office 365 workbench and get this errors 

**Warning!**
Use of this tool exposes you to potential security threats which can result in others gaining access to your personal Office 365 data (documents, emails, conversations and more). Make sure you trust the person or organization that asked you to access this tool before proceeding.

Learn more here: https://technet.microsoft.com/en-us/library/bb794823.aspx

VM1058:4 GET https://localhost:4321/temp/manifests.js net::ERR_CONNECTION_REFUSED
T @ VM1058:4
...
VM1058:4 Mixed Content: The page at 'https://.../sites/Developpersite/_layouts/15/workbench.aspx' was loaded over HTTPS, but requested an insecure XMLHttpRequest endpoint 'http://localhost:4321/temp/manifests.js'. This request has been blocked; the content must be served over HTTPS.
...
workbench-assembly_en-us_af4559a….js:150 Failed to load debug manifests: Error: Error loading http://localhost:4321/temp/manifests.js
	XHR error loading http://localhost:4321/temp/manifests.js
(anonymous) @ workbench-assembly_en-us_af4559a….js:150
workbench-assembly_en-us_af4559a….js:145 The Button component has been deprecated. Use specific variants instead. (PrimaryButton, DefaultButton, IconButton, CommandButton, etc.)
...
VM1058:4 Uncaught (in promise) Error: Error loading http://localhost:4321/temp/manifests.js
	XHR error loading http://localhost:4321/temp/manifests.js
    at XMLHttpRequest.o (eval at <anonymous> (workbench-assembly_en-us_af4559a….js:140), <anonymous>:4:14084)

![image](https://user-images.githubusercontent.com/6776269/26985623-ea1ffbac-4d76-11e7-93e7-8e3c8f890357.png)

#### Steps to Reproduce
Install the SharePoint framework using the tutorial 1.0.0
Configure the debug mode
![image](https://user-images.githubusercontent.com/6776269/26985636-ff349714-4d76-11e7-8a5f-ddadedad86b0.png)
Run the debug mode

#### To troubleshoot I have : 

- Update to SPFX 1.1.0
- Ran gulp untrust-dev-cert
- Ran gulp trust-dev-cert

#### Question : 
Do you know what is this warning "Use of this tool exposes you to potential security threats which ... " ??

For info : Chrome 59

**Comment by manishgarg1** [comment]
Hi @seaSide974, Do you see a shield icon at the right edge of the browser address bar? if yes, can yo utry and click on that to "load unsafe content" if that start working for you?

**Comment by patmill** [comment]
So the warning message is intentional, and shouldn't block anything.  It's to stop social engineering type attacks on people.  The weird thing sounds like you are getting blocked on https on your localhost.  
Does the same thing happen with different browsers?  Did you accidentally put a - in your gulp command when you ran it?

**Comment by seaSide974** [comment]
Thank you for your quick answer, I restarted evereything now i can debug, i have those error, i guess i can ignore it ?

![image](https://user-images.githubusercontent.com/6776269/27000418-24f54f4e-4de4-11e7-857a-ce10ba1ec1f0.png)
![image](https://user-images.githubusercontent.com/6776269/27000419-299c2dec-4de4-11e7-8d3e-a430ce533786.png)

**Comment by seaSide974** [comment]
Also i have uninstalled this extension in chrome that set https everywhere, maybe it was due to that ? I dont know https://chrome.google.com/webstore/detail/https-everywhere/gcbommkclmclpchllfjekcdonpmejbdp?hl=en

**Comment by iclanton** [comment]
Have you trusted the development certificate? You can run `gulp trust-dev-cert` and set the `https` property to `true` in the `serve.json` file.

**Comment by seaSide974** [comment]
As Mentioned i have already done gulp trust-dev-cert
I verified the serve.json, https is set to true

**Comment by iclanton** [comment]
Did this recently work? Do you have a firewall configured that blocks connections to localhost? Can you browse directly to `https://localhost:4321/`?

**Comment by nikunj231991** [comment]
In workbench i get this error but why i get after deploying to azure cdn?
I already done "gulp trust-dev-cert and set the https property to true in the serve.json file."
but still getting 


Warning!
Use of this tool exposes you to potential security threats which can result in others gaining access to your personal Office 365 data (documents, emails, conversations and more). Make sure you trust the person or organization that asked you to access this tool before proceeding.

Learn more here: https://technet.microsoft.com/en-us/library/bb794823.aspx

**Comment by iclanton** [comment]
Seems like this is resolved, so I'm going to close this issue. Please reopen if this is still an issue.

**Comment by tstojecki** [comment]
I think this needs a bit more clarification, this is pretty common in sp online.

When the message says "Use of this tool", what tool does it refer to exactly? @andrewconnell - are you saying that this message refers to the use of a browser console?

Also, it is one thing to display a warning (even if you decide to use a giant font size like this) and another to throw a whole bunch of errors. The errors are not being thrown to warn a user too, right?
